gitbase-web/templates/ingress.yaml

```diff
@@ -1,32 +1,39 @@
 {{- if .Values.ingress.enabled -}}
-{{- $serviceName := include "gitbase-web.fullname" . -}}
+{{- $fullName := include "gitbase-web.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
 {{- $servicePort := .Values.service.port -}}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ template "gitbase-web.fullname" . }}
+  name: {{ $fullName }}
   labels:
-    app: {{ template "gitbase-web.name" . }}
```

Comment:
Labels should be consistent across all templates. So, unless we change everything, I wouldn't touch this.

There are hundreds of charts using old-style labels, so I think it is not worth changing this

Reply:
Note that if you create new charts with `helm create` you will get new style labels. I assume we'll be using new style for new charts, rather than doing extra effort to use the old style everywhere, right?

Reply:
Yes, new charts will get new style as soon as any of the releases > 2.9.x get to a patch version > 0. For the moment, I see a lot of movement in helm and tons of pre-releases